requirements.txt

```diff
@@ -13,3 +13,4 @@ aiohttp<4,>=3.7.4
 eth-utils<1.10
 click
 importlib_resources>=5.2
```

Comment:
Oh, my bad, I forgot to remove this file. Please, remove it, it's not used for anything